octobot/src/assets/repos.html

```diff
@@ -45,7 +45,12 @@ <h4>Repo</h4>
           <h4>Git</h4>
           <div class="checkbox">
             <label>
-              <input type="checkbox" ng-model="theRepo.force_push_notify"> Force-push notification
+              <input type="checkbox" ng-model="theRepo.force_push_notify"/> Force-push notification
+            </label>
+          </div>
+          <div class="checkbox">
+            <label>
+              <input type="checkbox" ng-model="theRepo.use_threads"/> Use threads for PRs
```

Comment:
maybe connect this with slack for clarity? 

```suggestion
              <input type="checkbox" ng-model="theRepo.use_threads"/> Use slack threads
```